solutions/cses-1625.mdx

```diff
@@ -0,0 +1,349 @@
+---
+id: cses-1625
+source: CSES (TODO -- convert to something like `USACO Silver 2017 January`)
```

Comment:
reverted again? i think you forgot to `git pull` :/ some other changes were reverted too